proto/raft_serverpb.proto

```diff
@@ -117,10 +117,13 @@ enum ExtraMessageType {
     MsgHibernateRequest = 4;
     MsgHibernateResponse = 5;
     MsgRejectRaftLogCausedByMemoryUsage = 6;
+    MsgTracePeerAvailabilityInfo = 7;
```

Comment:
Can it be shorter?

Reply:
I don't think it can, I looked at synonyms and there is no short :(

Reply:
Does it need a request type and a response type?

Reply:
No, this message will be sent by non-leader peers when replying `MsgAppendResp` or `MsgHeartBeatResp`, leader use this type to collect infos.

Reply:
So it will be attached to every messages? I don't think it is necessary.

Reply:
Yes, as every `MsgAppendResp` as applied index changes, or `MsgHeartBeatResp` if there's long time no write.

Reply:
I don't object to report the response periodically, but I don't think it's necessary to be attached to heartbeat or append response. It should be queried on demand.

Reply:
If it's queried on demand, it's still triggered on leader periodically. Seems unnecessary

Reply:
Once it's recovered from not ready to ready and vice versa, then no need to query anymore until there the role is changed again.